src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -181,11 +191,11 @@ def _create_pre_defines(
         config_file_name = os.path.basename(config_file_path)
         config_file_basename = os.path.splitext(config_file_name)[0]
         return {
+            "<CONFIG_PATH>": config_file_dir,
+            "<CONFIG_FILE_BASE>": config_file_basename,
             "<DATE>": date_string,
             "<CWD>": config_file_dir,
-            "<CONFIG_PATH>": config_file_dir,
             "<CONFIG_FILE>": config_file_name,
-            "<CONFIG_FILE_BASE>": config_file_basename,
```

Comment:
please don't move the code just because OR sort it alphabetically

Reply:
This was part of making the two parsers finally result in the same order.